rnd/autogpt_server/autogpt_server/server/model.py

```diff
@@ -37,3 +40,13 @@ class SubscriptionDetails(pydantic.BaseModel):
     event_type: str
     channel: str
     graph_id: str
+
+
+class CreateGraph(pydantic.BaseModel):
+    template_id: str | None = None
+    template_version: int | None = None
+    graph: autogpt_server.data.graph.Graph | None = None
```

Comment:
Ideally this would be a discriminating model that accepts either set of attributes but not a combination.

Potential solutions:
* https://github.com/tiangolo/fastapi/pull/10773
* https://github.com/tiangolo/fastapi/discussions/9287#discussioncomment-5552717
* https://github.com/tiangolo/fastapi/issues/2279#issuecomment-787517707